Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -299,6 +301,7 @@ protected override FrameworkElement GenerateEditingElement(DataGridCell cell, ob
         protected override FrameworkElement GenerateElement(DataGridCell cell, object dataItem)
         {
             EnsureColumnBinding(dataItem);
+            this.EnsureColumnTypeAgreement(dataItem);
```

Comment:
nit: this.EnsureColumnTypeAgreement(dataItem); --> EnsureColumnTypeAgreement(dataItem);